package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "auth0-logs-to-mixpanel",
-
   "version": "2.0.1",
```

Comment:
need to bump version. given the dependency change, should probably be a minor bump to `2.1.0`.